scripts/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+DIR="$(dirname "$0")"
+
+# detect docker environment we're running in
+
```

Comment:
do we need to set the max memory for the JVM?
marathon seems to limit the max memory using `-Xmx512m` by default. this can be a problem if we do not override it, right?